packages/framework/src/Model/Product/ProductQuery.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Product;
+
+class ProductQuery
```

Comment:
Can we rename this to ProdutcCriteria please? :smile:

Reply:
I have renamed it to ProductQueryParams